src/models/EventMapper.php

```diff
@@ -453,6 +453,8 @@ public function transformResults($results, $verbose)
                             . $row['ID'] . '/comments/reported';
                         $list[$key]['reported_talk_comments_uri'] = $base . '/' . $version . '/events/'
                             . $row['ID'] . '/talk_comments/reported';
+                        $list[$key]['pending_claims_uri'] = $base . '/' . $version . '/events/'
```

Comment:
I agree that this would be preferable, let's not show stuff that the majority of people can't access